squire_sdk/src/server/accounts.rs

```diff
@@ -107,3 +110,18 @@ where
     state.load_user(user);
     StatusCode::ACCEPTED
 }
+
+pub async fn new_verification_data(key: String, user: User) -> VerificationData {
```

Comment:
There is no need for this to be `async`, and it should probably be a method called `new` for `VerificationData`. It also doesn't need to take a user. Associating a user with some verification data is the job of the server state, not the data itself. Please remove the user argument.

Reply:
alright, and yes i independently removed the user argument in my latest commit